app/javascript/components/components/Header.js

```diff
@@ -16,6 +16,16 @@ class Header extends Component {
       <>
         <h1>React in Rails with Devise</h1>
         <Nav>
+          {!logged_in &&
+            <NavItem>
+              <a href="/" className="nav-link">Home</a>
```

Comment:
Internal navigation should not have an `href` 
It should be a `NavLink` from react-router-dom